src/sensors/ErroneousSensor.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2022 TheDevMinerTV
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+*/
+
+#include "ErroneousSensor.h"
+#include "network/network.h"
+
+namespace SlimeVR
+{
+    namespace Sensors
+    {
+        void ErroneousSensor::motionSetup()
+        {
+            m_Logger.error("IMU of type %s failed to initialize", getIMUNameByType(m_ExpectedType));
+        }
+
+        void ErroneousSensor::sendData()
+        {
+            if (m_SentFirstData)
+            {
+                return;
+            }
+
+            Network::sendRotationData(&quaternion, DATA_TYPE_NORMAL, 255, sensorId);
```

Comment:
This is not what it needs to send te be displayed, it should just return `SensorStatus::SENSOR_ERROR` in `getSensorState()`, everything else will be handled.